content/docs/introduction/getting_started.md

```diff
@@ -86,6 +86,10 @@ Blindly setting `GOMAXPROCS` to a high value can be
 counterproductive. See the relevant [Go
 FAQs](http://golang.org/doc/faq#Why_no_multi_CPU).
 
+Notice that Prometheus by default uses around 3GB in memory, if you have smaller
```

Comment:
Start a new sentence after "memory" instead of using a comma (https://owl.english.purdue.edu/owl/resource/598/02/).

"smaller" -> "a smaller"